db-client/src/main/java/com/splicemachine/db/client/net/NetConfiguration.java

```diff
@@ -175,6 +175,9 @@
     // Indicates Kerberos authentication.
     public static final int SECMEC_KERSEC = 0x0B;
 
+    // Indicates Token authentication.
+    public static final int SECMEC_TOKEN = 0x0E;
```

Comment:
Did you come up with this value or have you taken it from somewhere? As far as I could see the DRDA security mechanism that would work here is PLGIN (Plug-in security) which has value `0x0F`. `0x0E` stands for `EUSRNPWDDTA` (Encrypted User ID, Encrypted Password, Encrypted New Password, and Encrypted Security-sensitive Data)

Reply:
Check https://splicemachine.atlassian.net/wiki/spaces/PD/pages/581369866/DRDA+Specification+Documents for the DRDA specs

Reply:
changed to use PLGIN. I've introduced the new constant 
`public static final int SECMEC_PLGIN = 0x0F;`
but still kept the old one with more meaning, that is used in the code:
`public static final int SECMEC_TOKEN = SECMEC_PLGIN;`